backend/src/main/java/com/skr/virtuallibrary/entities/enums/Genre.java

```diff
@@ -0,0 +1,7 @@
+package com.skr.virtuallibrary.entities.enums;
+
+public enum Genre {
+    ACTION,
+    HISTORICAL,
+    FANTASY
+}
```

Comment:
Enumy przychodzą z API jako integery czy stringi? Bo jeżeli jako integery, to nie byłoby możliwości wypisania nazwy gatunku.

Reply:
Przychodzą jako string